navigator/frontend/src/applets/contract/ContractComponent.tsx

```diff
@@ -52,8 +52,10 @@ interface ActiveLinkProps extends OwnProps {
   isActive: boolean;
 }
 
-const ActiveLink = (props: ActiveLinkProps) => (
-  <Link {...props} />
+const ActiveLink: React.FC<ActiveLinkProps> = (props) => (
```

Comment:
Without this, we get an error below where we try to do something like `<ActiveLink>…</ActiveLink>`. Not quite sure what got stricter but seems sensible to me.